test/parallel/test-child-process-send-serializer.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
```

Comment:
Not that it matters to much, but `assert` and `cp` could be required only in the parent.